scripts/build-lshpack.ps1

```diff
@@ -0,0 +1,18 @@
+$ErrorActionPreference = 'Stop'  # Setting strict mode, similar to 'set -euo pipefail' in bash
+. (Join-Path $PSScriptRoot "env.ps1")
+
+Push-Location (Join-Path $BUN_DEPS_DIR 'lshpack')
+try {
+  Set-Location (mkdir -Force build)
+  
+  Run cmake .. @CMAKE_FLAGS `
```

Comment:
you're sure this won't build DLL files and just a single static `.lib` file?

Reply:
should build a static file using `-DSHARED=0` will test on windows.